apps/opik-backend/src/main/java/com/comet/opik/domain/SpanDAO.java

```diff
@@ -170,6 +186,22 @@ INSERT INTO spans(
                     LENGTH(old_span.metadata) > 0, old_span.metadata,
                     new_span.metadata
                 ) as metadata,
+                multiIf(
+                    LENGTH(old_span.model) > 0, old_span.model,
+                    new_span.model
+                ) as model,
+                multiIf(
+                    LENGTH(old_span.provider) > 0, old_span.provider,
+                    new_span.provider
+                ) as provider,
+                multiIf(
+                    old_span.total_estimated_cost > toDecimal32(0, 7), old_span.total_estimated_cost,
```

Comment:
We don't need this, I think: toDecimal32(0, 7)

Reply:
Yep, works with just 0